app/models/game_odd.rb

```
@@ -27,4 +27,45 @@
 #  fk_rails_...  (game_id => games.id)
 #
 class GameOdd < ApplicationRecord
+  belongs_to :game
+  validates :game_id, uniqueness: true
```

RuboCop (1.75.5)

[convention] Uniqueness validation should have a unique index on the database column.

(Rails/UniqueValidationWithoutIndex)